client/src/components/payment/payment-modal.vue

```diff
@@ -113,6 +118,7 @@ export default class PaymentModal extends Mixins(
   }
 
   async confirmPayment () {
+    this.isLoadingPayment = true
```

Comment:
I think this needs to be set to False if the user cancels this modal.

Reply:
I think with vue modals, each time we close the modal, it's vue instance is destroyed and it's data along with it. Since this loading property is local, it should default back to `false` just by closing/cancelling it.

Reply:
Will test a few more times though, but so far it seems to be ok.

Reply:
But, actually now that you bring this up. Do you think we should disable cancel once they have initiated payment? 
If you click `Pay` but cancel immediately after, it still goes to pay.

Reply:
1. A modal is just a component. If the component was destroyed between instances then, yes, its data is cleared, and when it is re-instantiated then created() and mounted() are called, etc.

However, I think in this project, all modals are created once at app start and are never really destroyed, only hidden.

2. Definitely agree with you! Make it so. :)

Reply:
Hmm interesting, yea it seems like it is resetting somehow in the lifecycle of the modal, but i can certainly put in a safety reset!

Done and done 👍